src/cbl-mariner/manifest.json

```diff
@@ -20,7 +20,7 @@
             },
             {
               "architecture": "arm64",
-              "dockerfile": "src/cbl-mariner/2.0/docker-testrunner",
+              "dockerfile": "src/cbl-mariner/2.0/docker-testrunner/amd64",
```

Comment:
As you can see from the `architecture` property, this is `arm64` but is also used for `amd64` builds. In other words, this is a multi-arch Dockerfile. So it shouldn't be placed in an architecture-specific directory.

Reply:
Will fix - thanks!

Reply:
Fixed with https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1047/commits/8fce682c040a27b7d2ae70ef936d3b0f84434127